### ISSUE TITLE
operator CI build fails

### ISSUE THREAD

**Issue by gsunner** [SeldonIO/seldon-core#1330]
During the operator CI build, fails when doing the 'go vet'

```
go fmt ./...                                                                                                                                 
go vet ./...                                                                                                                                 
go: downloading github.com/onsi/ginkgo v1.8.0                                                                                                
go: downloading github.com/Azure/go-autorest/autorest v0.9.2                                                                                 
go: downloading sigs.k8s.io/testing_frameworks v0.1.1                                                                                        
go: downloading github.com/gophercloud/gophercloud v0.4.0                                                                                    
go: downloading github.com/onsi/gomega v1.5.0                                                                                                
go: extracting sigs.k8s.io/testing_frameworks v0.1.1                                                                                         
go: extracting github.com/Azure/go-autorest/autorest v0.9.2                                                                                  
go: extracting github.com/onsi/gomega v1.5.0                                                                                                 
go: downloading github.com/Azure/go-autorest/autorest/adal v0.7.0                                                                            
go: extracting github.com/onsi/ginkgo v1.8.0                                                                                                 
go: downloading github.com/Azure/go-autorest/logger v0.1.0         
go: downloading github.com/Azure/go-autorest/tracing v0.5.0 
go: extracting github.com/Azure/go-autorest/autorest/adal v0.7.0                                                                             
go: extracting github.com/Azure/go-autorest/logger v0.1.0                                                                                    
go: downloading github.com/dgrijalva/jwt-go v3.2.0+incompatible                                                                              
go: downloading github.com/Azure/go-autorest/autorest/date v0.2.0                                                                            
go: extracting github.com/Azure/go-autorest/tracing v0.5.0                                                                                   
go: extracting github.com/Azure/go-autorest/autorest/date v0.2.0
go: extracting github.com/dgrijalva/jwt-go v3.2.0+incompatible                                                                               
go: downloading github.com/hpcloud/tail v1.0.0          
go: extracting github.com/hpcloud/tail v1.0.0
go: downloading gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7
go: extracting gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7
go: extracting github.com/gophercloud/gophercloud v0.4.0
go: finding github.com/onsi/gomega v1.5.0
go: finding github.com/onsi/ginkgo v1.8.0                                                                                                    
go: finding sigs.k8s.io/testing_frameworks v0.1.1
go: finding github.com/Azure/go-autorest/autorest v0.9.2
go: finding github.com/gophercloud/gophercloud v0.4.0      
go: finding github.com/Azure/go-autorest/autorest/adal v0.7.0
go: finding github.com/hpcloud/tail v1.0.0                                                                                                   
go: finding github.com/Azure/go-autorest/tracing v0.5.0         
go: finding github.com/Azure/go-autorest/logger v0.1.0
go: finding github.com/Azure/go-autorest/autorest/date v0.2.0
go: finding github.com/dgrijalva/jwt-go v3.2.0+incompatible
go: finding gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7   
# k8s.io/client-go/rest                                                                                                                      
/builder/home/go/pkg/mod/k8s.io/client-go@v11.0.1-0.20190409021438-1a26190bd76a+incompatible/rest/request.go:598:31: not enough arguments in call to watch.NewStreamWatcher
        have (*versioned.Decoder)                                                                                                            
        want (watch.Decoder, watch.Reporter)                                                                                                 
Makefile:100: recipe for target 'vet' failed
```

**Comment by gsunner** [comment]
This error happend on the following pipeline
```
seldonio/seldon-core/master #141 release
```

**Comment by gsunner** [comment]
fixed in #1335
